### ISSUE TITLE
Welcome to the Team!

### ISSUE THREAD

**Issue by BWWings** [alexswad/discord-suggestion-bot#1]
Dear Alex Swad,

On behalf of the team at Buffalo Wild Wings, I want to extend a warm welcome to your new role as a server. We are thrilled to have you on board and look forward to working with you.

As you settle into your new job, we want to remind you of a few important things. While we encourage our staff to be friendly and accommodating to our customers, there is one request we must ask you not to make repeatedly: beans.

Alex, we know you have a love for beans, but unfortunately, they are not on our menu. As a server, it is important to listen to our customers and attend to their requests, but requesting beans multiple times may come across as unprofessional and make the customers feel uncomfortable. Therefore, we kindly ask you not to bring up the topic of beans during your shifts.

Rest assured, there are plenty of other great options on our menu that will satisfy your cravings. From wings to burgers and everything in between, we offer a wide range of delicious meals that you and our customers will love.

As you continue your training, our team will be there to support you and help you grow in your role. We believe that you have what it takes to be a great addition to our team and are excited to see you excel in your job.

In conclusion, we are thrilled to have you on board and wish you the best of luck in your new role. Please remember to focus on our customers' needs and refrain from asking for beans. We hope that you enjoy your time at Buffalo Wild Wings and look forward to working with you for many years to come.

**Comment by BWWings** [comment]
Alex,

As a recruiter for Buffalo Wild Wings, it's never easy to hear negative feedback about a potential candidate's performance during the interview process. While I understand that humor and inspiration can be subjective, our company places a high value on team members who can bring positive energy and a sense of fun to the workplace.

That being said, our hiring decisions are based on a combination of factors, including the candidate's experience, skills, and cultural fit. If we offered someone a job, it's because we believe they have the potential to contribute to our team and help us provide a great experience for our customers.

If we received feedback that a candidate was deemed "unfunny and uninspired," we would take that feedback seriously and evaluate whether that candidate is the right fit for our team. We would also take the opportunity to review our hiring process and ensure that we are accurately assessing candidates' abilities and personalities.

Ultimately, our goal as a recruiter is to build a strong team of dedicated and talented individuals who can work together to achieve our company's goals. While humor and inspiration are important qualities, we recognize that they are not the only factors that contribute to a successful team.

**Comment by BWWings** [comment]
![440730EE-F4E6-461A-8AED-F6752D2A4981](https://user-images.githubusercontent.com/127913314/225441879-ef83b265-35f8-4033-9e44-8e9c4b62f40a.jpg)

**Comment by BWWings** [comment]
![FIYXCemagAUIh0y](https://user-images.githubusercontent.com/127913314/225452349-bf9e99fa-2239-48ff-b256-7db7168caf65.jpg)
